<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="begin_pushbutton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Begin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="end_pushbutton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="undo_pushbutton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_pushbutton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="linear_radiobutton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>270</y>
     <width>191</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear Interpolation</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="vector_radiobutton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Vector Interpolation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_pushbutton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>380</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_pushbutton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>460</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>150</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="time_box">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>150</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QComboBox" name="speed_box">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>200</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Liner</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Accelerate</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Decelerate</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
